custom/icds_reports/static/icds_reports/js/spec/prevalence-of-stunting-report.directive.spec.js

```diff
@@ -14,25 +14,8 @@ describe('Prevalence Of Stunting Directive feature flag disable', function () {
     pageData.registerUrl('icds_locations', 'icds_locations');
 
     beforeEach(module('icdsApp', function ($provide) {
-        $provide.constant("genders", [
```

Comment:
next series of commits is just applying the same treatment to a bunch of other tests (which also gets them passing).